src/Umbraco.Core/Persistence/SqlSyntax/SqlServerSyntaxProvider.cs

```diff
@@ -250,6 +250,11 @@ public override bool DoesTableExist(IDatabase db, string tableName)
         }
 
         public override void WriteLock(IDatabase db, params int[] lockIds)
+        {
+            WriteLock(db, 1800, lockIds);
+        }
+
+        public void WriteLock(IDatabase db, int millisecondsTimeout, params int[] lockIds)
```

Comment:
This seems to be a breaking change..

Before `WriteLock(db, 42)` would hit `public override void WriteLock(IDatabase db, params int[] lockIds)` now it hits 
` public void WriteLock(IDatabase db, int millisecondsTimeout, params int[] lockIds)`

I think it is cleaner to change the signature to 
```
  public void WriteLock(IDatabase db, TimeSpan timeout, params int[] lockIds)
```

Reply:
Ah yes good catch, will fix up